src/main/java/ch/kalunight/zoe/repositories/RepoRessources.java

```diff
@@ -0,0 +1,50 @@
+package ch.kalunight.zoe.repositories;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import ch.kalunight.zoe.model.dto.DTO;
+import ch.kalunight.zoe.translation.LanguageManager;
+import net.dv8tion.jda.api.entities.MessageChannel;
+
+public class RepoRessources {

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'RepoRessources' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/KaluNight/Zoe-Discord-Bot/pullRequest?prid=4739924)